### ISSUE TITLE
Insufficient Contrast Between Background and Foreground Colors

### ISSUE THREAD

**Issue by saphirasnow** [CMU-313/Mayan-EDMS#2]
The Accessibility score reported by Google Lighthouse for the Mayan homepage is 89. One of the issues reported in this category is insufficient contrast between the background and foreground colors of the homepage's buttons.
